README.md

```diff
@@ -53,7 +53,7 @@ cd programs/tests/c-tests/fib/
 ../../../../scripts/c/riscv_build.sh fib fib.c
 
 # Run the simulation
-../../../../emulator/fp-emu +ispm=fib.hex
```

Comment:
Is there an actual format difference or just renaming the extension?

Reply:
No, there is no actual format difference. Renaming the extension to `.mem` enables direct import into Vivado to initialize the ISPM more easily.